### ISSUE TITLE
Windows 1.12.2 v1.0.0  Mobs do not Pathfind Around Windows

### ISSUE THREAD

**Issue by LivingLandmine** [sketchmacaw/MacawsModsIssues#264]
**Describe the bug**
Mobs cannot detect the collision of windows and will try to move through them, causing them to get stuck on them even when an unblocked path is available.

**Macaw's Windows 1.12.2 v1.0.0  Forge**

**To Reproduce**
1. Place wall of windows
2. Place a zombie or tamed wolf on the other side.
3. Have them attempt to pathfind to you.
4. They fail to go around the windows and instead stay stuck.

**Screenshots**
![2023-02-10_13 18 56](https://user-images.githubusercontent.com/31289215/218168608-02e1c6fb-3fa7-4eb5-8d56-51bab2dea471.png)

**Comment by LivingLandmine** [comment]
I've noticed its been a long time and there's been no changes or comments to this issue.  Is this because it's 1.12.2?  Just wondering why.

**Comment by sketchmacaw** [comment]
Yup, 1.12.2 hasn't been supported for some time, this issue is fixed in the newer versions. However I plan to make a last update for all 1.12.2 macaws mods.

**Comment by LivingLandmine** [comment]
Ah, cool, thank you.
